samples/msal-angular-v2-samples/angular10-browser-sample/README.md

```diff
@@ -1,8 +1,10 @@
-# Angular 10 MSAL.js 2.x Sample
+# Angular 10 MSAL-Browser Sample
 
 ## About this sample
 
-This developer sample is used to demonstrate how to use MSAL.js in Angular 10.
+This developer sample was created before `@azure/msal-angular@2` was available to demonstrate how to use `@azure/msal-browser` in an Angular application directly.
```

Comment:
Would this be more clear if this sample lives in the `msal-browser-samples` folder?

Reply:
After discussion with team, decided to leave this sample in the `msal-angular-v2-samples` folder for now.